content/articles/ansible-and-nix.md

```diff
@@ -0,0 +1,101 @@
+Title: My experience with Nix as an Ansible user
+Date: 2015-04-03
+Short_summary: I reflect on my experiences of using Ansible and Nix to deploy applications 
+Category: Infrastructure
+Authors: Gautier
+
+
+We recently deployed a Django application using a less known tool:
+[Nix](https://nixos.org/).
+
+Although I have tried Chef, Puppet and Salt Stack in the past I never settled
+for any of those tools.
+
+I've also «dockerized» a few applications but never used docker to run a
+production system because of the amount of work needed to manage the necessary
+production infrastructure to run docker containers.
+
+In this article I'm writing about my experience of configuring a new server
+with Nix given my experience with Ansible. I'm not getting into much details
+into either [Ansible](http://docs.ansible.com/) nor [Nix](https://nixos.org/)
+and focus mostly on my experience.
+
+## Ansible
+
+My favourite configuration management tool for the last couple of years has
+been Ansible. It helps for both server configuration, server orchestration and
```

Comment:
kill "both"  because you're mentioning 3 things

also is "helps" accurate? It "supports" maybe?